monkey/monkey_island/Pipfile.lock

```diff
@@ -164,14 +164,6 @@
             "markers": "python_version >= '3.7'",
             "version": "==8.1.3"
         },
-        "colorama": {
-            "hashes": [
-                "sha256:854bf444933e37f5824ae7bfc1e98d5bce2ebe4160d46b5edf346a89358e99da",
-                "sha256:e6c6b4334fc50988a639d9b98aa429a0b57da6e17b9a44f0451f930b6967b7a4"
-            ],
-            "markers": "platform_system == 'Windows'",
-            "version": "==0.4.5"
-        },
```

Comment:
Did you lock on Linux? You need to lock on Windows, otherwise packages get removed.

If you're just trying to install `egg-timer`, you can `pipenv install --keep-outdated egg-timer`, and that should also work (on Linux or Windows)